src/Features/Core/Portable/EditAndContinue/CompilationOutputs.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Reflection.PortableExecutable;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    /// <summary>
+    /// Reads compilation outputs such as output assembly and PDB.
+    /// </summary>
+    internal abstract class CompilationOutputs
```

Comment:
Moving this to `Microsoft.CodeAnalysis.Emit` namespace as it can be used for more than EnC.